src/dune_rules/ocaml_flags.ml

```diff
@@ -78,39 +102,54 @@ module Spec = struct
 
   let standard =
     { common = Ordered_set_lang.Unexpanded.standard
+    ; keywords = Keywords.standard
     ; specific = Lib_mode.Map.make_all Ordered_set_lang.Unexpanded.standard
     }
   ;;
 
-  let make ~common ~specific : t = { common; specific }
+  let make ~common ~keywords ~specific : t = { common; keywords; specific }
+
+  let decode_keywords =
+    let open Dune_lang.Decoder in
+    fields
+      (let+ version = field_o "version" string
```

Comment:
`ocamlc` enforces a rule that the version number be `<major>.<minor>`. I think it would be better UX for dune to also enforce this rule while parsing this field because otherwise malformed version strings will cause `ocamlc` to print its error, which won't have the context that dune has while parsing.